lib/crawlfs.js

```diff
@@ -20,11 +20,21 @@ module.exports = async function (dir, options) {
   const metadata = {}
   const crawled = await glob(dir, options)
   const results = await Promise.all(crawled.map(async filename => [filename, await determineFileType(filename)]))
+  const links = []
   const filenames = results.map(([filename, type]) => {
     if (type) {
       metadata[filename] = type
+      if (type.type === 'link') links.push(filename)
     }
     return filename
+  }).filter((filename) => {
+    // Newer glob can return files inside symlinked directories, to avoid
+    // those appearing in archives we need to manually exclude theme here
```

Comment:
can we turn this off in the glob options? looks like `{follow: false}` might do it.

Reply:
follow: false is the default, and I tried that initially 🤷